honeybee_energy/measure.py

```diff
@@ -0,0 +1,428 @@
+# coding=utf-8
+"""Module for parsing OpenStudio measures and setting measure arguments."""
+from __future__ import division
+
+import os
+import xml.etree.ElementTree as ElementTree
+
+
+class Measure(object):
+    """Object to hold all properties of an OpenStudio measure, including arguments.
+
+    Args:
+        folder: Path to the folder in which the measure exists. This folder
+            must contain a measure.rb and a measure.xml file. Other files are
+            optional.
+
+    Properties:
+        * folder
+        * xml_path
+        * rb_path
```

Comment:
`measure_path` might be a better option. `rb_path` gives a feeling that this is a ruby installation path to me.

I would also use `args_path` instead of `xml_path`. Basically what the files do rather than what is the file type.

Reply:
`measure_path` is what I would expect the root of the whole measure to be called. I'll pick something more descriptive, though.

Reply:
I went with `metadata_file` and `program_file` since this seems to be what they are called in the Measure Writers Guide.